src/Backend/Modules/Pages/Domain/ModuleExtras/ModuleExtra.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+namespace Backend\Modules\Pages\Domain\ModuleExtras;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity()
+ * @ORM\Table(name="modules_extras")
+ */
+final class ModuleExtra
+{
+    /**
+     * @var int
+     *
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     * @ORM\Column(type="integer", name="id")
+     */
+    private $id;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string", name="module")
+     */
+    private $module;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string", name="type")
```

Comment:
we already have `Common\ModuleExtraType` Maybe move it or something and make a DBAL type for it.
And then change it wherever it is used

Reply:
Changed in ad886fa2